_inc/lib/admin-pages/class.jetpack-my-jetpack-page.php

```diff
@@ -46,11 +46,34 @@ function jetpack_my_jetpack_change_user() {
 				Jetpack::init()->stat( 'admin', 'change-primary-successful' );
 				Jetpack::init()->do_stats( 'server_side' );
 
+				// Change the blog owner dotcom side
+				$this->tell_dotcom_about_user_switch( $new_master_user );
 			}
 		}
 	}
 
 	/*
+	 * Tell wpcom that the master user has switched
+	 * so we can update the 'wpcom_blog_owner'
+	 */
+	function tell_dotcom_about_user_switch( $new_master ) {
+		$id = Jetpack::get_option( 'id', false );
+		$wpcom_user = Jetpack::get_connected_user_data( $new_master );
+
+		$request = array(
+			'blog_id'        => $id,
+			'new_blog_owner' => $wpcom_user['ID']
+		);
+
+		// Tell wpcom about the change
+		Jetpack::load_xml_rpc_client();
+		$xml = new Jetpack_IXR_Client( array(
+			'user_id' => get_current_user_id()
```

Comment:
Should have a comma after the array index.  Also, should this query be made as the master user?  (I'm not sure)